docs/data/date-pickers/date-time-field/DateTimeFieldValue.tsx

```diff
@@ -10,11 +10,7 @@ export default function DateTimeFieldValue() {
 
   return (
     <LocalizationProvider dateAdapter={AdapterDayjs}>
-      <Stack
-        spacing={{ xs: 4, xl: 2 }}
-        direction={{ xs: 'column', xl: 'row' }}
-        sx={{ '& > *': { width: 300 } }}
-      >
+      <DemoContainer>
```

Comment:
After the changes the meridiem is being cut-off.
![Screenshot 2022-12-09 at 15 12 50](https://user-images.githubusercontent.com/4941090/206710287-3cce662a-5d53-47c7-8d6d-cb4d8c9e97f7.png)

Reply:
See https://github.com/mui/mui-x/pull/7149#discussion_r1045490074